rust/src/lib/nm/query_apply/ip.rs

```diff
@@ -74,6 +74,7 @@ pub(crate) fn nm_ip_setting_to_nmstate4(
             } else {
                 None
             },
+            forwarding: nm_forwarding_to_nmstate(nm_ip_setting),
```

Comment:
NM should not provide this value, nispor should. If you are querying this from NM, then the verification is doing nothing.

Reply:
Thanks for pointing it out, I dropped the code.